src/security/x-powered-by.conf

```diff
@@ -18,6 +18,8 @@
 #
 # https://php.net/manual/en/ini.core.php#ini.expose-php
 
+
+# Note you need both below as the "always" option doesn't always work with some servers for some reasons.
```

Comment:
This comment is no longer relevant, you can remove it as much as the line above. 😉

Reply:
Gah silly leftover, fixed. :P